RecoTracker/TransientTrackingRecHit/interface/Traj2TrackHits.h

```diff
@@ -17,10 +17,17 @@
 
 class Traj2TrackHits {
 private:
-  const StripClusterParameterEstimator * theCPE;
-  bool keepOrder;  // FIXME move to enum
-  bool removeNoDet;  // true == as in conversion from TTRH tp TRH
+  const StripClusterParameterEstimator * theCPE =nullptr;
+  bool keepOrder=false;  // FIXME move to enum
+  bool removeNoDet=true;  // true == as in conversion from TTRH to TRH
 public:
+  using TrajParams = std::vector<LocalTrajectoryParameters>;
+  using Chi2sFive = std::vector<unsigned char>;
+
+  static unsigned char toChi2(float chi2) { int tc = 5.f*chi2; return std::min(tc,255);}  
```

Comment:
and eventually the rounding can be improved...
I will look for a more general float -> fixed point conversion implementation..